.github/CODEOWNERS

```diff
@@ -2,32 +2,32 @@
 packages/react-chat/ @microsoft/teams-prg
 packages/react-data-grid-react-window/ @microsoft/teams-prg
 packages/react-shadow/ @microsoft/teams-prg
-packages/nx-plugin/ @microsoft/teams-prg
+packages/nx-plugin/ @microsoft/teams-prg @microsoft/fluentui-react-build
 packages/react-data-grid-react-window-grid/ @microsoft/ms-fabric
 packages/react-draggable-dialog/ @microsoft/cxe-prg @marcosmoura
 
 #### Build/Infra
-/.github/ @microsoft/teams-prg
-/tsconfig.base.json @microsoft/teams-prg
-/yarn.lock @microsoft/teams-prg
-/beachball.config.js @microsoft/teams-prg
-/beachball.hooks.js @microsoft/teams-prg
-/babel.config.json @microsoft/teams-prg
-/jest.config.ts @microsoft/teams-prg
-/jest.preset.js @microsoft/teams-prg
-/nx.json @microsoft/teams-prg
-/.gitignore @microsoft/teams-prg
-/.prettierignore @microsoft/teams-prg
-/.prettierrc @microsoft/teams-prg
-/.editorconfig @microsoft/teams-prg
-/.eslintignore @microsoft/teams-prg
-/.eslintrc.json @microsoft/teams-prg
-/.yarnrc @microsoft/teams-prg
-/.yarn/ @microsoft/teams-prg
+/.github/ @microsoft/teams-prg @microsoft/fluentui-react-build
```

Comment:
As you're adding v-build, please remove `@microsoft/teams-prg` from infra files

Reply:
makes sense ty!